src/main/java/org/sonarlint/intellij/core/BackendService.kt

```diff
@@ -103,24 +112,22 @@ class BackendService @NonInjectable constructor(private val backend: SonarLintBa
             serverConnections.filter { !it.isSonarCloud }.map { toSonarQubeBackendConnection(it) }
         backend.initialize(
             InitializeParams(
-                HostInfoDto(ApplicationInfo.getInstance().versionName),
-                TelemetryManagerProvider.TELEMETRY_PRODUCT_KEY,
+                ClientInfoDto(
+                    ApplicationInfo.getInstance().versionName,
+                    TelemetryManagerProvider.TELEMETRY_PRODUCT_KEY,
+                    "SonarLint IntelliJ " + getService(SonarLintPlugin::class.java).version
+                ),
+                FeatureFlagsDto(true, true, true, true, true),
```

Comment:
Aren't TaintVulnerabilities disabled in Rider or CLion?

Reply:
The corresponding flag is deprecated and should be removed. The CORE has the knowledge of when to download taint vulnerabilities or not

Reply:
*deprecated and not used